dpgen/generator/run.py

```diff
@@ -2158,10 +2169,10 @@ def run_fp (iter_index,
             mdata) :
     fp_style = jdata['fp_style']
     fp_pp_files = jdata['fp_pp_files']
-
+    backward_files  = ['fp.log']
```

Comment:
Note that only VASP uses the default `log_file`. Others set it to `output`.
https://github.com/deepmodeling/dpgen/blob/d22316e71a4ae0b2cd8246b83771931e2bd778e4/dpgen/generator/run.py#L2190

Reply:
Done